src/test/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinterTest.java

```diff
@@ -78,7 +79,7 @@ public void testParseFileWithError() throws Exception {
     @Test
     public void testCreationOfFakeCommentBlock() throws Exception {
         final Method createFakeBlockComment =
-                Whitebox.getMethod(DetailNodeTreeStringPrinter.class,
+                Whitebox.getMethod(JavadocUtils.class,
```

Comment:
after move to `JavadocUtils` method `createFakeBlockComment` can be called directly